domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewService.kt

```diff
@@ -157,8 +158,10 @@ class ReviewService(
       SentenceType.REMAND -> ReviewScheduleCalculationRule.PRISONER_ON_REMAND
       SentenceType.CONVICTED_UNSENTENCED -> ReviewScheduleCalculationRule.PRISONER_UN_SENTENCED
       SentenceType.INDETERMINATE_SENTENCE -> ReviewScheduleCalculationRule.INDETERMINATE_SENTENCE
+      SentenceType.SENTENCED, SentenceType.RECALL -> reviewScheduleCalculationRuleBasedOnTimeLeftToServe(releaseDate!!)
 
-      else -> reviewScheduleCalculationRuleBasedOnTimeLeftToServe(releaseDate!!)
+      SentenceType.DEAD, SentenceType.CIVIL_PRISONER, SentenceType.IMMIGRATION_DETAINEE, SentenceType.UNKNOWN, SentenceType.OTHER ->
```

Comment:
There are certain sentence types that we dont support in respect of Reviews / Review Schedules